apps/contributor/package.json

```diff
@@ -7,10 +7,10 @@
     "@aragon/cli": "^5.9.6"
   },
   "devDependencies": {
-    "@aragon/test-helpers": "^1.1.0",
     "eth-gas-reporter": "^0.2.0",
     "ganache-cli": "^6.4.3",
-    "solidity-coverage": "^0.5.11"
+    "solidity-coverage": "^0.5.11",
+    "@aragon/test-helpers": "^1.0.0"
```

Comment:
do we still need those? and if so should we use the latest version?

Reply:
I used the test-helpers package in the tests. For the solidity-coverage we may need it... or should I remove it ?

Reply:
no, I was only wondering about the test-helpers. can we use the latest version here? 2.0.0
https://www.npmjs.com/package/@aragon/test-helpers or does that beak things?

Reply:
I updated it now, did not break anything :)